src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -39,6 +39,12 @@ message Input {
   optional google.datastore.v1.Entity variant = 8;
   optional string original_job_type = 9;
   optional string fuzzer_name = 10;
+  optional int64 crash_revision = 11;
```

Comment:
This is getting a bit long and complicated. Can we make this more structured by defining sub-types?

Reply:
Done